lib/test/cdo/test_web_purify.rb

```diff
@@ -0,0 +1,29 @@
+require_relative '../../../shared/test/test_helper'
+require_relative '../../cdo/pegasus'
+require_relative '../../cdo/web_purify'
+require 'minitest/autorun'
+
+class WebPurifyTest < Minitest::Test
+  include SetupTest
+
+  # Do additional VCR configuration so as to prevent the CDO.webpurify_key from being logged to the
+  # YML cassette, instead replacing it with a placeholder string.
+  VCR.configure do |c|
+    c.filter_sensitive_data('<API_KEY>') {CDO.webpurify_key}
+  end
```

Comment:
Depending on how paranoid we want to be, we could do this preemptively in `shared/test/test_helper`, iterating over all CDO keys.